transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionExtension.java

```diff
@@ -64,7 +65,8 @@
     static final ModelVersion MODEL_VERSION_EAP62 = ModelVersion.create(1, 3);
     static final ModelVersion MODEL_VERSION_EAP63 = ModelVersion.create(1, 4);
     static final ModelVersion MODEL_VERSION_EAP64 = ModelVersion.create(1, 5);
-    private static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(3, 0, 0);
+    static final ModelVersion MODEL_VERSION_WFLY10 = ModelVersion.create(3, 0);
+    private static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(3, 1, 0);
```

Comment:
The general rule is no model version bumps for an EAP micro. There may of course be special considerations I am not aware of.